components/builder-web/app/package/package-sidebar/package-sidebar.component.ts

```diff
@@ -51,6 +51,12 @@ export class PackageSidebarComponent {
            this.isNewProject ? 'Build latest versions' : `Build latest ${this.platform.name} version`;
   }
 
+  get buildButtonDisabledMessage() {
+    return this.targetIsMac ?
+      `* Builder can't build the package because a macOS plan file is not supported yet. Stay tuned.` :
```

Comment:
I don't know if I would say "Stay tuned."... might be tuned for a very long time :)

Reply:
Ah lol, well I just went with the text included in the mockup. I can drop the 'stay tuned' or change the message entirely. @apriofrost @susanev can chime in if they have any opinions on alternative messaging.

<img width="214" alt="Screen Shot 2019-09-23 at 3 43 27 PM" src="https://user-images.githubusercontent.com/479121/65465232-facf9400-de18-11e9-8d6d-448385b2352f.png">

Reply:
@chefsalim yeah. I agree. Let's drop the "Stay tunned".